.circleci/config.yml

```diff
@@ -8,6 +8,9 @@ jobs:
     docker:
       # specify the version you desire here
       - image: circleci/clojure:lein-2.9.3-buster
+      - image: localstack/localstack
```

Comment:
Doe this mean you are replacing the circleci/clojure image here with the other one?

Reply:
Nope, this is how you make your job use multiple images.

https://circleci.com/docs/2.0/executor-types/#using-multiple-docker-images